test-suite-data.json

```diff
@@ -252,7 +252,7 @@
     "is_invalid": false
   },
   {
-    "description": "slash /// after type  is not significant",
+    "description": "leading slash /// in front of type  is not significant",
```

Comment:
I think it would be slightly more accurate to say that "trailing slash after `scheme` should be removed" since it's more associated with that field than the schema.

Reply:
or perhaps "all trailing slashes after `scheme` should be removed" ?  BTW, do these descriptions need to refer to both the scheme and the colon ':' separator, since the slashes follow not just the scheme but the scheme and its colon separator?  Or is that unneeded detail?

Reply:
I used `leading` (or I suppose we could use preceding) because of the colon separator. If we said "trailing slash after schema" I think that would apply to `pkg////:`

Reply:
That's how I'd interpret it as well.  :+1:

Reply:
Ha, in `pkg////:` I understand `pkg` is the scheme field, `////` are trailing slashes, and `:` is the separator between scheme and type :-) The separators, in my view, don't really belong to any of the components.